requirements.txt

```diff
@@ -63,8 +63,8 @@ requests==2.25.1
 requests-futures==1.0.0
 requests-mock==1.8.0
 requests-toolbelt==0.9.1
--e git+https://github.com/WIPACrepo/rest-tools@d07c78c0978f6f581e50db89a156a3103a2f2fb9#egg=rest_tools
-setuptools==53.0.0
+-e git+https://github.com/WIPACrepo/rest-tools@c85209680c1ee6de031f4b0d93db7b2145a9bf0f#egg=rest_tools
```

Comment:
using the release tag (`@1.0.4`) may more manageable. I've been wanting to get into the habit using this syntax

Reply:
instead of the commit hash

Reply:
That doesn't seem to work. Looks like some problem in the setup.py script for rest-tools?
```
Obtaining rest_tools from git+https://github.com/WIPACrepo/rest-tools@1.0.4#egg=rest_tools (from -r requirements.txt (line 66))
  Updating ./env/src/rest-tools clone (to revision 1.0.4)
  Running command git fetch -q --tags
  Running command git reset --hard -q 54cc6b913f44fd1c4678c57ae0aac6f5e2519f2d
    ERROR: Command errored out with exit status 1:
     command: /home/pmeade/github/lta-dev/env/bin/python3.8 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/home/pmeade/github/lta-dev/env/src/rest-tools/setup.py'"'"'; __file__='"'"'/home/pmeade/github/lta-dev/env/src/rest-tools/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-8uu1ig3f
         cwd: /home/pmeade/github/lta-dev/env/src/rest-tools/
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/home/pmeade/github/lta-dev/env/src/rest-tools/setup.py", line 30, in <module>
        kwargs['version'] = line.split('=')[-1].split('\'')[1]
    IndexError: list index out of range
    ----------------------------------------
WARNING: Discarding git+https://github.com/WIPACrepo/rest-tools@1.0.4#egg=rest_tools. Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

But using the commit hash works fine:
```
Obtaining rest_tools from git+https://github.com/WIPACrepo/rest-tools@0557e76b37955efdecb980110c65b98b99577500#egg=rest_tools (from -r requirements.txt (line 66))
  Updating ./env/src/rest-tools clone (to revision 0557e76b37955efdecb980110c65b98b99577500)
  Running command git fetch -q --tags
  Running command git rev-parse -q --verify 'sha^0557e76b37955efdecb980110c65b98b99577500'
  Running command git fetch -q https://github.com/WIPACrepo/rest-tools 0557e76b37955efdecb980110c65b98b99577500
  Running command git reset --hard -q 0557e76b37955efdecb980110c65b98b99577500
```

Reply:
That's an interesting error.  I'm unsure if it's opening the `__init__.py` file in the right directory?

Reply:
oh I know the problem

Reply:
I fixed that yesterday, but I guess it didn't make the release

Reply:
@blinkdog try now with 1.0.5

Reply:
That worked!
```
Obtaining rest_tools from git+https://github.com/WIPACrepo/rest-tools@1.0.5#egg=rest_tools (from -r requirements.txt (line 66))
  Updating ./env/src/rest-tools clone (to revision 1.0.5)
  Running command git fetch -q --tags
  Running command git reset --hard -q 67952735d3f488266e04f512514563020269bc05
```